node_interop/test/promises_test.dart

```diff
@@ -56,6 +56,14 @@ void main() {
     expect(promiseToFuture(promise2), completion('YesYesYes'));
   });
 
+  test('futureToPromise', () {
+    final JsPromises js = require(promises);
+    var future = Future.value('Yes');
+    var promise = futureToPromise(future);
+    var promise2 = js.receivePromise(promise);
+    expect(promiseToFuture(promise2), completion('YesYesYes'));
+  });
+
```

Comment:
This looks like a duplicate of a test case from above?

Reply:
Oops, that's a leftover from a change I ended up not making.